package.json

```diff
@@ -0,0 +1,63 @@
+{
+	"name": "price-calculator",
+	"version": "0.0.1",
+	"description": "ArDrive Price Calculator",
+	"scripts": {
+		"start": "snowpack dev",
+		"build": "yarn clean && snowpack build",
+		"test": "web-test-runner \"src/**/*.test.{ts,tsx}\"",
+		"format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
+		"lint": "eslint --fix .",
+		"ci": "yarn test && yarn build",
+		"clean": "rimraf build"
+	},
+	"license": "AGPL-3.0-or-later",
```

Comment:
If this is the license we're going to go with, we should add a LICENSE.md to the repo

Reply:
Its in the repo: https://github.com/ardriveapp/price-calculator/blob/PE-89/react-app-setup/LICENSE

I copy-pasted the one from ardrive-core-js in [this commit](https://github.com/ardriveapp/price-calculator/commit/b911cf5194be4f93e903079aaf23eac6a344eab9)